searchcore/src/vespa/searchcore/proton/server/documentdb.cpp

```diff
@@ -222,16 +235,14 @@ void DocumentDB::registerReference()
     }
 }
 
-void DocumentDB::setActiveConfig(const DocumentDBConfig::SP &config,
-                                 SerialNum serialNum, int64_t generation) {
+void DocumentDB::setActiveConfig(const DocumentDBConfig::SP &config, int64_t generation) {
     lock_guard guard(_configMutex);
     registerReference();
     _activeConfigSnapshot = config;
     assert(generation >= config->getGeneration());
     if (_activeConfigSnapshotGeneration < generation) {
         _activeConfigSnapshotGeneration = generation;
     }
-    _activeConfigSnapshotSerialNum = serialNum;
```

Comment:
Why is this removed?

Reply:
Some compilers warn about private member variables that appears to be unused. The variable is never read by the C++
code, so the only use might be indirectly (e.g. when inspecting core dumps).

Reply:
I think that having variables around for future possible coredumps is not a good pattern.